src/cata_tiles.cpp

```diff
@@ -2572,7 +2572,7 @@ bool cata_tiles::draw_vpart( const tripoint &p, lit_level ll, int &height_3d,
     const bool overridden = override != vpart_override.end();
     // first memorize the actual vpart
     const optional_vpart_position vp = g->m.veh_at( p );
-    if( vp && !invisible[0] ) {
+    if( vp && ( &vp->vehicle() == g->remoteveh_cache || !invisible[0] ) ) {
```

Comment:
Does this mean you can see every part of a remote controlled vehicle even if you cannot normally see them? If that's the case it's not very realistic. People should only be able to see their remotely controlled vehicle if they actually installed a camera system to monitor the parts they want to see.

Reply:
As for me, it is same as expirienced driver can "feel" the dimensions of vehicle, even he can't see  some of its parts.

Reply:
Also it's a sort of tradeof between "playability" and realism.

Reply:
An experienced drive may feel the dimension of the vehicle, but the current code also lets one know any change that happens to the vehicle telepathically. It's not that realistic if your remotely controlled vehicle bumps into something and you know immediately it's the left headlight that got knocked off.

Reply:
You can develop the ability to "feel" the dimensions of the vehicle in game, no need to arbitrarally make the vehicle count as transparent.